tests/Services/CQS/CommandBuses/SynchronousTest.php

```diff
@@ -55,28 +41,11 @@ public function findCommandHandlerFor(Command $command): CommandHandler
         };
     }
 
-    private function buildRefusedCommandCommandHandler()
-    {
-        return new Class implements CommandHandler {
-            public function accept(Command $command): bool
-            {
-                return false;
-            }
-            public function handle(Command $command): void
-            {
-            }
-        };
-    }
-
-    private function buildAcceptedCommandCommandHandler()
+    private function spyCommandHandler()
     {
         return new Class implements CommandHandler {
```

Comment:
Now there is a  `ClosureBased` CommandHandler that can be used here to avoid an anonymous class.

Reply:
Fixed 13d8615